packages/web-react/src/components/Heading/demo/HeadingHyphens.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { TextHyphens } from '../../../constants';
+import Heading from '../Heading';
+
+const HeadingWordBreak = () => (
+  <div style={{ maxWidth: '200px' }}>
+    <Heading elementType="h2" size="xsmall">
+      Default text with no specific hyphenation
```

Comment:
maybe this should also have a long word? So we can see how it behaves without any class